sphinx/source/operators/start_network.rst

```diff
@@ -18,7 +18,8 @@ To create a new CCF network, the first node of the network should be started wit
     --enclave-type debug
     --node-address node_ip:node_port
     --rpc-address <ccf-node-address>
-    --public-rpc-address public_rpc_ip:public_rpc_port
+    --public-rpc-address <ccf-node-public-address>
+    [--domain domain]
```

Comment:
does it make sense to `s/domain/node-fqdn` or mention in the docs that this is node's FQDN to make more explicit?

i was initially confused with `domain` as network's domain, that can accept wild-card characters like `*.blockchian.azure.com`. 

p.s sorry for post-merge comments